service/controller/app/resource/chartoperator/resource.go

```diff
@@ -20,7 +25,8 @@ import (
 )
 
 const (
-	Name = "chartoperator"
+	Name                             = "chartoperator"
+	AppOperatorTriggerReconciliation = "app-operator.giantswarm.io/trigger-reconciliation"
```

Comment:
Should we move it to the https://github.com/giantswarm/k8smetadata/blob/be2bc263815cd4b8a8a76ed35ca1b15ab1cc9120/pkg/annotation/app.go? Currently I am setting a date as a value, so maybe `triggered-reconciliation-at` would be more appropriate name?

Reply:
I think the name is fine but moving it to k8smetadata makes sense to me.

Reply:
Yes let's add it there. I think `triggered-reconciliation` is fine. We just need the update event so we don't care about the value. Although the timestamp is a good idea for debugging. 👍

Reply:
Thanks guys, I'm introducing this new annotation here: https://github.com/giantswarm/k8smetadata/pull/32